src/HealthcareApis/HealthcareApis/HealthcareApis.csproj

```diff
@@ -8,7 +8,7 @@
   <Import Project="$(MSBuildThisFileDirectory)..\..\Az.props" />
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.HealthcareApis" Version="1.1.0" />
+    <PackageReference Include="Microsoft.Azure.Management.HealthcareApis" Version="3.0.0" />
   </ItemGroup>
```

Comment:
Just curious if this should be 1.2.0 to match what is in the ChangeLog.md file? Or are they unrelated versions?

Reply:
Hmm I think making the change log 3.0.0 might make sense, will check to see what Ankita did with PL

Reply:
Updated change log to be equal to this